manifests/exception.pp

```diff
@@ -133,16 +136,11 @@
     }
 
     # Validate common parameters
-    validate_re($ensure,['^(present|absent)$'])
     validate_slength($display_name,255)
```

Comment:
You could set the type of ```$display_name``` to ```String[0, 255]``` to limit it.

Reply:
This, yes. But I actually didn't really want to touch too many of the  params that I didn't touch in my initial request :-)

Reply:
Sure.  If you've got time, sure, if not, your PR is still an improvement and much appreciated.